ExpenseTracker.Core/Models/Transactions/Exceptions/FailedTransactionStorageException.cs

```diff
@@ -15,9 +15,11 @@ public class FailedTransactionStorageException : Xeption
 
         public FailedTransactionStorageException(Exception innerException)
             : base(message: "Failed transaction storage error occurred, contact support.", innerException)
-        {
+        { }
 
-        }
+        public FailedTransactionStorageException(string message, Exception innerException)
+            : base(message: "Failed transaction storage error occurred, contact support.", innerException)
+        { }
 
         public FailedTransactionStorageException(SqlException sqlException)
```

Comment:
Remove this constructor it does not belong here.

Reply:
@glhays I am using this constructor for DBUpdateException shown below. Shall I keep it or remove it?

![image](https://github.com/pratikkaje/ExpenseTracker.Core/assets/88729747/4b3af2de-7219-45db-88de-295b863cd8e9)

Reply:
Like I said above.
That is the function of your.
        private async ValueTask<Transaction> TryCatch(ReturningTransactionFunction returningTransactionFunction). 
If your implentation is not catching the exception, there is something wrong elsewhere.

Reply:
It seems the intention here is to verify the message during the localization process of the DbUpdateException. If a developer decides to alter the exception message, the test will detect this change and result in failure. I also referred intertrack project and they also have the constructor as shown below.

![image](https://github.com/pratikkaje/ExpenseTracker.Core/assets/88729747/8c8fa21a-b8e7-47e9-9e9d-dae9370f0a6d)

Reply:
@pratikkaje 
I do not see this constructor in theirs?????
```
public FailedTransactionStorageException(SqlException sqlException)
{
            this.sqlException = sqlException;
}
Quote reply

Reply:
And yes your are correct about your "seems the intention is"

Reply:
@glhays Thank you for your feedback. I believe I now grasp the issue. I have proceeded to remove the constructor mentioned below. Please correct me if I am still missing the point. 😅

public FailedTransactionStorageException(SqlException sqlException)
{
            this.sqlException = sqlException;
}